core/storage/base_model/gae_models_test.py

```diff
@@ -137,6 +143,43 @@ class TestVersionedModel(base_models.VersionedModel):
     SNAPSHOT_CONTENT_CLASS = TestSnapshotContentModel
 
 
+class BaseCommitLogEntryModelTests(test_utils.GenericTestBase):
+
+    def test_base_class_get_instance_id_raises_not_implemented_error(self):
+        # Raise NotImplementedError as _get_instance_id is to be overwritten
+        # in child classes of BaseCommitLogEntryModel.
+        with self.assertRaises(NotImplementedError):
+            base_models.BaseCommitLogEntryModel.get_commit('id', 1)
+
+
+class BaseSnapshotMetadataModelTests(test_utils.GenericTestBase):
+
+    def test_get_version_string(self):
+        model1 = base_models.BaseSnapshotMetadataModel(
+            id='model_id-1', committer_id='committer_id', commit_type='create')
+        model1.put()
+        self.assertEqual(model1.get_version_string(), '1')
+
+    def test_get_unversioned_instance_id(self):
+        model1 = base_models.BaseSnapshotMetadataModel(
+            id='model_id-1', committer_id='committer_id', commit_type='create')
+        model1.put()
+        self.assertEqual(model1.get_unversioned_instance_id(), 'model_id')
```

Comment:
Add a check for what `get_version_string` will return/raise here and elsewhere.

Reply:
Um I also wrote a test explicitly for get_version_string(test_get_version_string) just above it. It tests for version strings with the same model properties

Reply:
I see, I was confused that a model could have a versioned vs unversioned state. I see that's wrong, so nvm this comment!